castai/resource_autoscaler.go

```diff
@@ -235,6 +236,22 @@ func resourceAutoscaler() *schema.Resource {
 											},
 										},
 									},
+									FieldPodPinner: {
+										Type:        schema.TypeList,
+										Optional:    true,
+										MaxItems:    1,
+										Description: "defines the Cast AI Pod Pinner components settings.",
```

Comment:
```suggestion
										Description: "defines the Cast AI Pod Pinner components settings. Default enabled.",
```

Reply:
Added default state to the `pod_pinner.enabled` field description.